packages/notebook/style/notebookfooter.css

```
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+.jp-Notebook-footer {
+  height: 10%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

This will need some changes as soon as the widget is a button.

```suggestion
  display: flex;
  justify-content: center;
  align-items: center;
```